### PR TITLE
Remove gpseginstall

### PR DESCRIPTION
**Pull request by jmcatamney**
Now that the enterprise version of GPDB is only provided via RPM, including
gpseginstall in the distribution would cause conflicts if users try to install
GPDB with RPMs and with gpseginstall on the same cluster.  While it could be
preserved for use by the OSS community, there are several standard tools for
copying GPDB files to segment hosts in a cluster, and recommendations for using
one or more of those tools will be included in the GPDB documentation.

In addition to removing gpseginstall itself, this commit removes references to
it in other utilities' documentation and removes code in gppylib that was only
called by gpseginstall.

### CODE REVIEW

File: gpMgmt/bin/gppylib/commands/base.py [hunk 1]
```diff
@@ -472,190 +464,19 @@ def execute(self, cmd, wait=True):
...
-    SFTP_GET = 2
-
-    def __init__(self, passwordMap, sftp_operation=SFTP_NONE, sftp_remote=None, sftp_local=None):
-        self.passwordMap = passwordMap
-        self.sftp_operation = sftp_operation
-        self.sftp_remote = sftp_remote
-        self.sftp_local = sftp_local
-
-
-class NakedExecutionPasswordMap:
```
Comment (dkrieger-pvtl) [inline gpMgmt/bin/gppylib/commands/base.py]: Deleting this function should allow you to also remove the line `import paramiko, getpass` from this file as well.  I would think an auto importer cleaner would have done that.  Please do so, and clean up any other unused imports.

**Comment by dyozie** [comment]
Just want to confirm - is the intent to remove this in both 6.x (next Beta release), or is the removal only intended for 7.x?

File: gpMgmt/bin/gppylib/commands/base.py (hunk 1 above)
Comment (jmcatamney) [reply gpMgmt/bin/gppylib/commands/base.py]: Good catch, will do.

**Comment by jmcatamney** [comment]
@dyozie Yes, gpseginstall will be removed in 6.x, we'll be putting up a PR for that as soon as this one goes into master.

File: gpMgmt/bin/gppylib/commands/base.py (hunk 1 above)
Comment (jmcatamney) [reply gpMgmt/bin/gppylib/commands/base.py]: @dkrieger-pvtl Fixed those and force-pushed, and pylint didn't pick up any other unused imports.

**Pull request merged**